### ISSUE TITLE
seeing `Error: Missformed sytem command` errors in the 1.1 release build

### ISSUE THREAD

**Issue by chenrui333** [chatziko/lci#16]
👋 looks like the `lci-1.1.tar.gz` is missing for 1.1 release tag, so I tweak the homebrew build setup a bit like this

```diff
 class Lci < Formula
   desc "Interpreter for the lambda calculus"
   homepage "https://www.chatzi.org/lci/"
-  url "https://github.com/chatziko/lci/releases/download/v1.0/lci-1.0.tar.gz"
-  sha256 "1bcf40d738ce2af7ca5116f02dfb0f4ed21d7e467e3618e071c8199a1285331e"
+  url "https://github.com/chatziko/lci/archive/refs/tags/v1.1.tar.gz"
+  sha256 "bcfc8680b9ee47230aa1f6a9951d2f093ca6b0d752f9e67db91b1bc20a9ee197"
   license "GPL-2.0-or-later"

   bottle do
@@ -24,13 +24,26 @@ class Lci < Formula

   conflicts_with "lolcode", because: "both install `lci` binaries"

+  resource "dparser" do
+    url "https://github.com/jplevyak/dparser/archive/refs/tags/v1.33a.tar.gz"
+    sha256 "327a4783327d6dfe115fa4b85513fb36ff3ccbb17e027ee79c8a78c42099a73e"
+  end
+
+  resource "replxx" do
+    url "https://github.com/AmokHuginnsson/replxx/archive/refs/tags/release-0.0.4.tar.gz"
+    sha256 "a22988b2184e1d256e2d111b5749e16ffb1accbf757c7b248226d73c426844c4"
+  end
+
   def install
+    (buildpath/"dparser").install resource("dparser")
+    (buildpath/"replxx").install resource("replxx")
+
     system "cmake", "-S", ".", "-B", "build", *std_cmake_args
     system "cmake", "--build", "build"
     system "cmake", "--install", "build"
   end
```

The source build went out fine, but seeing some "Error: Missformed sytem command" when running `lci`, not sure if I missed anything. Thanks!

Here are some detailed prints

```sh
$ /opt/homebrew/Cellar/lci/1.1/bin/lci
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
Error: Missformed sytem command. Type Help for info.
lci - A lambda calculus interpreter
Type a term, Help for info or Quit to exit.

lci> Help

lci - A lambda calculus interpreter

Type a lambda term to compute its normal form
or enter one of the following system commands:

FixedPoint		Removes recursion using fixed point comb. Y
DefOp name prec ass	Defines an operator
ShowAlias [name]	List the specified or all stored aliases
Print term		Displays the term
Consult file		Reads and interprets the specified file
Set option (on|off)	Changes one of the following options:
			trace, showexec, showpar, greeklambda, readable
Help			Displays this message
Quit			Quit the program (same as Ctrl-D)
```

---

Context is I am trying to help with the brew formulate update.

**Comment by chatziko** [comment]
Thanks for reporting this, I added a source release with all dependencies, note that it's called `lci-v1.1.tar.gz` (not `lci-1.1.tar.gz`) to be consistent with the release tags.

Can you try building from there?

(PS. The error indicates a parsing issue, not sure what's happening exactly)

**Comment by chenrui333** [comment]
> Thanks for reporting this, I added a source release with all dependencies, note that it's called `lci-v1.1.tar.gz` (not `lci-1.1.tar.gz`) to be consistent with the release tags.
> 
> Can you try building from there?

totally, gonna try out that release artifact now

> 
> (PS. The error indicates a parsing issue, not sure what's happening exactly)

yeah, not sure either

----

https://github.com/Homebrew/homebrew-core/pull/201260

**Comment by chenrui333** [comment]
we shipped lci 1.1, closing this issue for now.
